bcos-boostssl/websocket/WsService.cpp

```diff
@@ -75,11 +75,18 @@ void WsService::start()
     // start as client
     if (m_config->asClient())
     {
-        if (m_config->connectedPeers() && !m_config->connectedPeers()->empty())
+        if (m_config->connectPeers() && !m_config->connectPeers()->empty())
         {
-            // Connect to peers and wait for at least one connection to be successfully established
+            for (auto connectPeer : *m_config->connectPeers())
+            {
+                WEBSOCKET_SERVICE(INFO) << ("#### start as client 0000")
```

Comment:
log for debug be should removed or canonical log printing

Reply:
fixed